coc.md

```diff
@@ -0,0 +1,106 @@
+
+Not all spam posts are by spam users. For example, a legitimate user account may be compromised. If a moderator notices a reputable user account posting spam, the spam post(s) should be deleted immediately and the user notified. If spam posts continue, the account may be suspended until the user can regain control of his/her account.
+
+Commercial content is allowed if relevant to answering a question or keeping inline with a topic.
+
+#### Reporting guidelines
+If someone is subject to or witnesses unacceptable behavior, or has any other concerns, they are strongly encouraged to notify an organizer, moderator or a GB Leadership team member as soon as possible. Every reported concern will follow this workflow until resolved:
+
+- Make contact by directly talking to an organizer, moderator or emailing support@gobridge.org
+  - Your email message will reach the **GB Leadership team** only.
+  - Reports will be confidential within the organizers team, moderators team or GB’s Leadership team.
+
+- You will receive a response within 24-48 hours (very likely sooner) in our online communities. For our offline communities and events, a response should be expected within an hour or two.
+- The group will review the incident and determine what happened.
+- They may reach out to other community members for more context.
+- The team will reach a decision as to how to act. These include:
+  - Nothing.
+  - A private reprimand from the GB Leadership team to the individual(s) involved.
+  - An imposed vacation (i.e. asking someone to "take a week off" from all GB moderated properties).
+  - A request for a private or public apology.
+  - A temporary or permanent ban from some or all GB moderated communities and events. GB Leadership team will reach out to the original reporter to let them know what action we decided to take.
+  - Appeals to the decision may be made to the GB Leadership team directly, or any member of the administration team thereof.
+
+#### Administrators
+Below is the list of people who are moderators for our online communities. If you have any issues or need help, please feel free to reach out to anyone on this list. There is also the public `admin-help` channel on Slack where people can get help or clarification on any admin related issues.
+
+**The role of GB (Leadership Team):**
+- They are the stewards for the different online communities.
+- They have rights in the systems tied to the primary owner role.
+- They act as administrators.
+
+**The role of moderators:**
+- They monitor the public forums, channels and threads for inappropriate behavior.
+- They have full voting rights for decisions that affect the communities.
+- They have rights in the systems tied to the admin role.
+
+**Primary Owner:**  
+GB (support@gobridge.org)
+
+**GB Leadership Team:**  
+[Ashley McNamara](https://twitter.com/ashleymcnamara),
```

Comment:
Let's list carolyn, maartje and rashmi here.

Reply:
The WWG leadership team is still showing the GB leadership team.

Reply:
Sure, Carolyn I'll work on this; couldn't find time to work on it in midst of office-work so thought of completing on Saturday. Hope that's okay? :)

Reply:
Of course, that's fine! Never any need to explain that this is volunteer work and real work, life, etc always come first.